### DISCUSSION TITLE
[BUG] Bode plot channel volts/div

### DISCUSSION THREAD

**Discussion by norbertkiszka** [mriscoc/Sparrow-Extended-UI-for-RIGOL-DHO800_DHO900#7] [General]
First try to do bode plot made a app crash. After restart, no waveform view and most of buttons doesn't work. After pressing "default" twice, it went back to normal - at least it look like.

Now, every time, no matter of settings (channels, acquisition, etc), after starting bode plot, last channel (no matter if this is input or output) changes to 200 uV/div... And this obviously makes bode plot not usable at all.

**Comment by norbertkiszka** [comment]
I flashed another SD card with my earliest backup from this scope with original app and turns out, bode plot is working correctly only on channels 1 and 2. After installing modified app, right now it works exactly the same - only channel 1 and 2.

**Comment by norbertkiszka** [comment]
I made a video with this bug: https://www.youtube.com/watch?v=KZtuv9nD4lc

**Reply by mriscoc** [reply]
Unfortunately, such bugs must be resolved by Rigol in a firmware update, we don't have the appropriate sources to make a fix.
